pkg/bsql/query.go

```diff
@@ -359,6 +359,21 @@ func (s *SQLSyncer) prepareQueryVars(ctx context.Context, inputs map[string]any,
 		inputs = make(map[string]any)
 	}
 
+	celInputs := make(map[string]any)
```

Comment:
So if inputs["inputs"] doesn't exist, we want to copy everything in the inputs map to inputs["inputs"]? If something is calling prepareQueryVars() with the wrong map, why not fix the call site to construct the map correctly?